sign_chck.c

```diff
@@ -0,0 +1,22 @@
+/*Sean Kee*/
```

Comment:
You actually did a harder assignment! You only needed to have your program check if the number was negative or positive! With that said, your code was perfect for the task you did do!